pyp5js/fs.py

```diff
@@ -1,17 +1,22 @@
 import os
+import re
 import shutil
 from pathlib import Path
 from cprint import cprint
 
 from pyp5js.config import SKETCHBOOK_DIR
-from pyp5js.exceptions import SketchDirAlreadyExistException
+from pyp5js.exceptions import SketchDirAlreadyExistException, InvalidName
 
 
 class SketchFiles():
     TARGET_NAME = 'target'
     STATIC_NAME = 'static'
 
     def __init__(self, sketch_name):
+        does_not_start_with_letter = r'^[^a-zA-Z]'
+        contains_non_alphanumerics = r'[^a-zA-Z0-9]'
```

Comment:
"_" is a valid char and is missing in this regexp.

Reply:
Going to add that @turicas ! Thanks for the reply

Reply:
+1 for the `_` here